test.properties

```diff
@@ -3211,6 +3253,292 @@
             (testray.main.component.name == "User Tracker")\
         )
 
+    #
+    # Core Infrastructure Acceptance
+    #
+
+    modules.includes[modules-semantic-versioning][core-acceptance]=${core.infra.testing.modules}
+
+    modules.includes.public[modules-semantic-versioning][core-acceptance]=${core.infra.testing.modules}
+
+    test.batch.class.names.includes[modules-integration-aws-s3-mysql57][core-acceptance]=\
+        **/apps/change-tracking/change-tracking-store-test/**/*Test.java,\
+        **/apps/portal-store/**/*Test.java
+
+    test.batch.class.names.includes[modules-integration-*][core-acceptance]=\
+        **/apps/bean-portlet/**/*Test.java,\
+        **/apps/change-tracking/change-tracking-store-test/**/*Test.java,\
+        **/apps/company/**/*Test.java,\
+        **/apps/counter/**/*Test.java,\
+        **/apps/diff/**/*Test.java,\
+        **/apps/expando/**/*Test.java,\
+        **/apps/external-reference/**/*Test.java,\
+        **/apps/gogo-shell/**/*Test.java,\
+        **/apps/imageio-plugins/**/*Test.java,\
+        **/apps/json-storage/**/*Test.java,\
+        **/apps/learn/**/*Test.java,\
+        **/apps/license-manager/**/*Test.java,\
+        **/apps/mail/**/*Test.java,\
+        **/apps/normalizer/**/*Test.java,\
+        **/apps/portal/**/*Test.java,\
```

Comment:
@kevhlee Not all the modules in apps/portal are in core infra scope, for example db partition, this needs to be listed in details

Reply:
@tinatian, Ok I will try to be more granular with the `apps/portal` modules and make sure to remove the out-of-scope ones.